engine/lite/kafka/runtime/src/main/scala/pl/touk/nussknacker/engine/lite/kafka/NuKafkaRuntimeApp.scala

```diff
@@ -26,16 +26,24 @@ object NuKafkaRuntimeApp extends App with LazyLogging {
 
   val scenario = parseScenario
 
-  val scenarioInterpreter: KafkaTransactionalScenarioInterpreter = prepareScenarioInterpreter
+  val runtimeConfig = ConfigFactory.load(ConfigFactoryExt.parseUnresolved(classLoader = getClass.getClassLoader))
 
+  val system = ActorSystem("nu-kafka-runtime", runtimeConfig)
+  import system.dispatcher
+
+  val scenarioInterpreter = prepareScenarioInterpreter(runtimeConfig)
   Runtime.getRuntime.addShutdownHook(new Thread() {
     override def run(): Unit = {
       logger.info("Closing KafkaTransactionalScenarioInterpreter")
       scenarioInterpreter.close()
```

Comment:
should we close also the ActorSystem here?

Reply:
Good point, added